### ISSUE TITLE
Log website url on MarkBind deploy

### ISSUE THREAD

**Issue by amad-person** [MarkBind/markbind#844]
Currently, the following message is shown on a successful deploy:

```
  __  __                  _      ____    _               _ 
 |  \/  |   __ _   _ __  | | __ | __ )  (_)  _ __     __| |
 | |\/| |  / _` | | '__| | |/ / |  _ \  | | | '_ \   / _` |
 | |  | | | (_| | | |    |   <  | |_) | | | | | | | | (_| |
 |_|  |_|  \__,_| |_|    |_|\_\ |____/  |_| |_| |_|  \__,_|
                                                           
 v2.3.0
info: Deployed!
```

Modifying the log message to `info: Deployed at <URL>` would allow users to navigate to the deployed site quickly. 

Relevant code:
https://github.com/MarkBind/markbind/blob/2a5e64572ad43cdf4d7a1ef7d41452c9c9f4d171/index.js#L196-L199

**Comment by nbriannl** [comment]
Hi, I'm trying to start out on this project so I'm picking up this issue. If anyone's already on it, do tell me! Thanks.

**Comment by nbriannl** [comment]
Hi! If I understand correctly, `markbind deploy` command happens when you deploy to Github Pages as said in the [User Guide](https://markbind.org/userGuide/deployingTheSite.html), is there any need to show the deployment url when deploying via Travis CI or Netifly?

**Comment by KendrickAng** [comment]
Hi @ang-zeyu , could I take this issue as a first issue?

**Comment by ang-zeyu** [comment]
yes, sure @KendrickAng. You might want to pick up where #947 left off as well if possible!

**Comment by KendrickAng** [comment]
I'll take a look at it, thanks!

**Comment by KendrickAng** [comment]
@ang-zeyu I believe this issue can be closed, or is there some policy we have regarding `good-first-issues`?

**Comment by ang-zeyu** [comment]
resolved by #1329
